### ISSUE TITLE
Remove start timer button - trigger timer start when starting annotation

### ISSUE THREAD

**Issue by laurentletg** [neuropoly/slicer-manual-annotation#57]
**Summary**
- Remove the Start Timer button - re : extra step that is not required. 
- Annotation time recording could start when the user pushes paint or erase, or switch to the segment editor module. 

**Describe the solution you'd like**
- Call the start timer method from appropriate methods or buttons/signals.

**Comment by AcastaPaloma** [comment]
would it be wise to create a new section called ''configure segmentation'' and include ''configure labels'' within this alongside the options about the timer? it would be better if we present the user with the option as to when the segmentation starts (paint push or segmentation chosen).

**Comment by delphinepilon** [comment]
It could be done. The button "Configure labels..." could be renamed to "Configure Segmentation" and the option to display the timer could be included in the label configuration window, above the label table view. If this was the case, the option to include the semi-automatic PHE segmentation tool should also be moved to this window. 

However, these changes should be included in another issue. It could even be discussed whether they should be done within the current pull request (#56) for issue #33.

**Comment by AcastaPaloma** [comment]
For users that choose to still display the timer, the start button would still be useful for them? If not, it is easily removable.

**Comment by maxradx** [comment]
yes! maybe it would be useful to have the start timer button available only if display timer is selected in the configuration file?
